proposals/PROPOSALS.md

```diff
@@ -0,0 +1,156 @@
+A HIP (*Haskell Foundation Initiative Proposal*) is a process for submitting a proposal for an initiative to be undertaken by the Haskell Foundation (HF). The main motivation of this document is to detail the primary mechanism through which the community may propose, discuss and implement initiatives to be carried out through the Haskell Foundation that will affect the Haskell ecosystem. For each HIP, all changes to the ecosystem go through a proposal process, which are is discussed by the HF Technical Track, the broader community, and any and all interested stakeholders. Only upon reaching a consensus are initiatives accepted to be merged and executed.
```

Comment:
(worksmithing)

"For each HIP, all changes to the ecosystem go through a proposal process, which ..."  is hard to parse.  Maybe

Each HIP goes through an iterative process in which it is discussed by the HF Technical Track, the broader community, and any and all interested stakeholders.    If a consensus is reached, the initiative is accepted and may then be executed by the Foundation (resources permitting).

Reply:
Love it.